packages/ihe-gateway/server/CodeTemplates/Common Library/Write document to AWS S3 bucket/Write document to AWS S3 bucket.js

```diff
@@ -15,14 +15,21 @@ function xcaWriteToFile(path, documentContents, metadata) {
 
     // Specify file's metadata
     var meta = java.util.HashMap();
+    var contentType = "application/octet-stream"; // Default content type
     for (var key in metadata) {
-      if ("url" !== key.toString()) meta.put(key.toString(), String(metadata[key]));
+      if ("url" !== key.toString()) {
+        meta.put(key.toString(), String(metadata[key]));
+        if (key.toString() === "contentType") {
+          contentType = String(metadata[key]);
+        }
```

Comment:
NABD w/ the current options, but I'd have the if for content type outside of the if to url - better for maintenance, less likely to bug later if we need to change this.